src/dataflow-types/src/logging.rs

```diff
@@ -137,8 +137,10 @@ impl LogVariant {
             LogVariant::Materialized(MaterializedLog::SourceInfo) => RelationDesc::empty()
                 .with_column("source_name", ScalarType::String.nullable(false))
                 .with_column("source_id", ScalarType::String.nullable(false))
+                .with_column("dataflow_id", ScalarType::String.nullable(false))
                 .with_column("partition_id", ScalarType::String.nullable(false))
                 .with_column("offset", ScalarType::Int64.nullable(false))
+                .with_column("timestamp", ScalarType::Int64.nullable(false))
                 .with_key(vec![0, 1, 2]),
```

Comment:
this line indicates that we should treat the first, second and third feels as constituting a primary key on this relation, but thats no longer true (because a single source can have multiple partitions).

you can either edit this line to be `vec![0, 1, 3]` or reposition dataflow id

sorry I know this system is extremely brittle